addons/popochiu/engine/objects/gui/components/dialog_text/dialog_overhead/dialog_overhead.gd

```diff
@@ -27,13 +26,13 @@ func _set_default_label_size(lbl: Label) -> void:
 
 
 func _append_text(msg: String, props: Dictionary) -> void:
+	msg = _correct_line_breaks(msg)
+	
```

Comment:
Assign the `[color][/color]` to the `msg` in the same way you did for the `dialog_caption.gd:17` and the `dialog_portrait.gd:47`.